Dockerfile

```diff
@@ -16,15 +16,16 @@ RUN apt-get -y update; apt-get -y install curl default-jre-headless unzip findut
 RUN python -m venv /venv
 ENV PATH="/venv/bin:$PATH"
 
+RUN curl -LO https://github.com/kaitai-io/kaitai_struct_compiler/releases/download/0.10/kaitai-struct-compiler-0.10.zip
```

Comment:
putting this on top will result in more efficient images and faster build times (now the `kaitai_struct_compiler` is not downloaded/unzipped each time a python requirement was changed or a struct was updated)